README.md

```diff
@@ -2,12 +2,25 @@
 
 Ubuntu is a popular Linux distribution based on Debian and composed mostly of free and open-source software that runs from the desktop to the cloud, to all your Internet connected things. It is the world's most popular operating system across public clouds and OpenStack clouds.
 
-wikipedia.org/wiki/Ubuntu
+[Ubuntu description on Wikipedia](https://wikipedia.org/wiki/Ubuntu)
```

Comment:
This is a mere style and all Makejails should have this style, don't worry about tagging the one that sends you to the project description.